.github/tool_checkmake.sh

```
@@ -0,0 +1,78 @@
+#    shall THE AUTHOR's total liability to you for all damages (other than as may
+#    be required by applicable law in cases involving personal injury) exceed
+#    the amount of five dollars ($5.00). The foregoing limitations will apply
+#    even if the above stated remedy fails of its essential purpose.
+################################################################################
+#
+# .github/tool_checkmake.sh
+FILE="${1}" ;
+EMSG="Checkmake linter complained.";
+
+# Check if file exists
+if [[ !( -f "${FILE}" ) ]]; then
```

Shellcheck

[error] You are missing a required space here.

(SC1035)